src/Customer/Service.php

```diff
@@ -17,4 +18,42 @@ interface Service
      * @return Customer
      */
     public function create(Token $token, string $email, string $description): Customer;
+
+    /**
+     * Retrieves a customer.
+     *
+     * @param string $uid
+     *
+     * @return Customer
+     */
+    public function find(string $uid): Customer;
+
+    /**
+     * Associates a card to a customer.
+     *
+     * @param Customer $customer
+     * @param Card $card
+     *
+     * @return Card
+     */
+    public function addCard(Customer $customer, Card $card): Card;
+
+    /**
+     * Deletes a customers card.
+     *
+     * @param Customer $customer
+     * @param Card $card
+     *
+     * @return void
+     */
+    public function deleteCard(Customer $customer, Card $card): void;
+
+    /**
+     * Retrieves a customers cards.
+     *
+     * @param Customer $customer
+     *
+     * @return Card[]
+     */
+    public function getCards(Customer $customer): array;
```

Comment:
can't have `Card[]` as the return type here?

Reply:
Unfortunately nope I'm getting an error if I'm trying.